src/app/integrations/intacct/intacct-shared/intacct-connection-form/intacct-connection-form.component.html

```
@@ -0,0 +1,32 @@
+         <div [ngClass]="isReconnecting ? '' : 'tw-p-24-px'">
+           <div [ngClass]="isReconnecting ? '' : 'container'">
+             <div class="tw-mb-16-px">
+               <p for="companyId" class="tw-text-form-label-text-color tw-text-14-px tw-pb-8-px">{{ 'intacctConnector.companyIdLabel' | transloco }}<app-mandatory-field [noSpacing]="true"></app-mandatory-field></p>
+               <input [placeholder]="'intacctConnector.enterCompanyIdPlaceholder' | transloco: { companyId: 'intacct.common.companyId' | transloco }" type="text" id="companyId" name="companyId" formControlName="companyID" class="custom-focus tw-border tw-border-gray-300 tw-rounded tw-px-3 tw-py-2" [ngClass]="[connectIntacctForm.controls.companyID.invalid && connectIntacctForm.controls.companyID.touched ? 'error-box' : 'normal-box', isReconnecting ? 'tw-w-full' : 'tw-w-400-px']">
+               <div *ngIf="connectIntacctForm.controls.companyID.invalid && connectIntacctForm.controls.companyID.touched" class="incorrect-credential tw-pt-2-px">
+                 {{ 'intacctConnector.invalidCompanyIdError' | transloco: { companyId: 'intacct.common.companyId' | transloco } }}
+               </div>
+             </div>
+             <div class="tw-mb-16-px">
+              <p class="tw-text-form-label-text-color tw-text-14-px tw-pb-8-px tw-pt-16-px">{{ 'intacctConnector.userIdLabel' | transloco }}<app-mandatory-field [noSpacing]="true"></app-mandatory-field></p>
+              <input [placeholder]="'intacctConnector.enterUserIdPlaceholder' | transloco: { userId: 'intacct.common.userId' | transloco }" type="text" id="userId" name="userId" formControlName="userID" [ngClass]="isReconnecting ? '' : 'tw-w-400-px'" class="custom-focus tw-border tw-border-gray-300 tw-rounded tw-px-3 tw-py-2" [ngClass]="[connectIntacctForm.controls.userID.invalid && connectIntacctForm.controls.userID.touched ? 'error-box' : 'normal-box', isReconnecting ? 'tw-w-full' : 'tw-w-400-px']">
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ [ngClass] ] was found.

(attr-no-duplication)